CFC_DataCollector/recommender/utility_model_pipeline.py

```diff
@@ -0,0 +1,17 @@
+"""
+Construct user utility model or retrieve from database and update with
+augmented trips. Store in database and return the model.
+"""
+
+from user_utility_model import UserUtilityModel
+
+def build_user_model(user_id, augmented_trips):
```

Comment:
who is going to pass in the augmented_trips? You still need to query for them from the database. Are you just going to query for all trips, or do you want to have some kind of filter (e.g. only trips in the past 6 mos). May want to add a select trips step.